src/app/components/common/Button/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,19 @@
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import React from 'react';
```

Comment:
Import React should be very first import in any tsx file

Reply:
done